### DISCUSSION TITLE
Offline Encyclopedia of Endless OS

### DISCUSSION THREAD

**Discussion by HomeroBretas** [community/community#121818] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

I would like to ask the community here to contact the "Endless OS Foundation LLC" and ask them to update the company's proprietary "Offline Encyclopedia" program for the Brazilian public. I've contacted members of the Endless forum several times, but Endless won't update this software for anything in the world. Please contact the operating system Encyclopedia developers of "Endless OS Foundation LLC" and ask them to update the Operating System, along with the Software for the Brazilian public! Thank you very much! Note: I am asking this community for help because I have no knowledge of software development and operating systems. But I need the Endless system as an end user, as I am an educator.

**Comment by ebndev** [comment]
Hi @HomeroBretas,

Thanks for posting in the Community! Unfortunately, we currently do not allow self-promotion or solicitation. We want to make sure there is space for users to ask questions without overwhelming them with other conversations. Thank you for helping us maintain a productive and tidy community for all our members.
